docs/usage.md

```diff
@@ -124,6 +124,46 @@ Refer to the API reference documentation of this library for more information ab
 | `getPresetInstrumentations` | [API reference](./api-reference/functions/getPresetInstrumentations.md) |
 | `getAioRuntimeResource`     | [API reference](./api-reference/functions/getAioRuntimeResource.md)     |
 
+### Integrations
+
+Since version 1.1.0, this library supports an `integrations` feature that provides pre-configured patches for external systems. Integrations handle complex setup tasks like context propagation, span linking, and sampling decisions automatically.
```

Comment:
```suggestion
Since version 1.1.0, this library supports an `integrations` feature that provides preconfigured patches for external systems. Integrations handle complex setup tasks like context propagation, span linking, and sampling decisions automatically.
```